package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "lambdatest-cypress-cli",
-  "version": "3.0.12",
+  "version": "3.0.13",
   "description": "The lambdatest-cypress-cli is LambdaTest's command-line interface (CLI) aimed to help you run your Cypress tests on LambdaTest platform.",
```

Comment:
action item:
- lets update DB and then do sanity post cli installation

Reply:
performed sanity on stage.